http/src/request/channel/message/allowed_mentions.rs

```diff
@@ -130,6 +134,21 @@ impl<'a> Default for AllowedMentionsBuilder<'a, Unspecified, Unspecified, Unspec
     }
 }
 
+impl<'a, E, U, R> AllowedMentionsBuilder<'a, E, U, R> {
+    /// Whether to mention the user being replied to.
+    ///
+    /// Defaults to false.
```

Comment:
The docs were corrected, and the default is actually false: https://github.com/discord/discord-api-docs/commit/a533e5d48e2f6d60ba1451c6835ec5b9bb0c69bd

We should likely match this behaviour

Reply:
The commit in got reverted with the reasoning here https://github.com/discord/discord-api-docs/pull/2246#issuecomment-730806702.

Reply:
this is still wrongly documented as per the comment linked above